src/Tests/AssemblyDigger_Tests.cs

```diff
@@ -13,7 +13,9 @@ public void Can_Dig_Assembly_Refs_By_Name()
 			var digger = new AssemblyDigger(Assembly.Load("ExceptionReporter.NET"));
 			var refs = digger.GetAssemblyRefs().ToList();
 
-			Assert.That(refs.Select(r => r.Name), Is.SupersetOf(new [] {"System.Core", "DotNetZip", "SimpleMapi.NET"}));
+			//TODO: is "System.Core" necessesary here?
+			//Assert.That(refs.Select(r => r.Name), Is.SupersetOf(new [] {"System.Core", "DotNetZip", "SimpleMapi.NET"}));//
```

Comment:
Yeah no, `System.Core` isn't necessary here.
This is just a slightly dodgy "integration" test I did... testing for DLLs I knew should be there, at the time. We can just remove that check